Gems/EditorPythonBindings/Code/Source/ActionManager/PythonActionManagerHandler.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Source/ActionManager/PythonActionManagerHandler.h>
+
+#include <AzToolsFramework/ActionManager/Action/ActionManagerInterface.h>
+#include <Source/PythonCommon.h>
+#include <pybind11/pybind11.h>
+
+PythonEditorActionHandler::PythonEditorActionHandler()
+{
+    m_actionManagerInterface = AZ::Interface<AzToolsFramework::ActionManagerInterface>::Get();
+    AZ_Assert(m_actionManagerInterface, "Could not get ActionManagerInterface on PythonEditorActionHandler construction.");
+
+    EditorPythonBindings::CustomTypeBindingNotificationBus::Handler::BusConnect(azrtti_typeid<PythonEditorAction>());
+    ActionManagerRequestBus::Handler::BusConnect();
+}
+
+PythonEditorActionHandler ::~PythonEditorActionHandler()
+{
+    ActionManagerRequestBus::Handler::BusDisconnect();
+    EditorPythonBindings::CustomTypeBindingNotificationBus::Handler::BusDisconnect();
+}
+
+AzToolsFramework::ActionManagerOperationResult PythonEditorActionHandler::RegisterAction(
+    const AZStd::string& contextIdentifier,
+    const AZStd::string& identifier,
+    const AZStd::string& name,
+    const AZStd::string& description,
+    const AZStd::string& category,
+    const AZStd::string& iconPath,
+    PythonEditorAction handler)
+{
+    return m_actionManagerInterface->RegisterAction(
+        contextIdentifier, identifier, name, description, category, iconPath,
+        [h = AZStd::move(handler)]()
+        {
+            PyObject_CallObject(h.GetHandler(), NULL);
```

Comment:
Can we use `nullptr` here instead of `NULL`?

Reply:
I think so, I was mostly just following the guidelines for the Python functions.